src/SixLabors.Fonts/Tables/General/ColrTable.cs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Linq;
+using SixLabors.Fonts.Tables.General.Colr;
+using SixLabors.Fonts.Tables.General.Glyphs;
+
+namespace SixLabors.Fonts.Tables.General
+{
+    [TableName(TableName)]
```

Comment:
What are these attributes used for?

Reply:
the are used in the `TableLoader` class [here](https://github.com/SixLabors/Fonts/blob/f05fd390be8b276d4cc982aef8ec6274a144d6fc/src/SixLabors.Fonts/Tables/TableLoader.cs#L22) to help link a table loader class with a static load method to a font table name so the loader can easily lookup the stream offsets required for accessing the table data.